x-pack/solutions/observability/plugins/observability_ai_assistant_app/docs/openapi/observability_ai_assistant_app_apis.yaml

```
@@ -0,0 +1,264 @@
+        class_name:
+          type: string
+          description: The class name associated with the entity.
+        start_pos:
+          type: number
+          description: The starting position of the unredacted entity.
+        end_pos:
+          type: number
+          description: The ending position of the unredacted entity.
+        type:
+          type: string
+          description: The type of unredaction applied.
+          enum:
+            - ner
+            - regex
+      required:
+        - entity
+        - class_name
+        - start_pos
+        - end_pos
+        - type
+    MessageRoleEnum:
+      description: The role of the message sender.
+      type: string
+      enum:
+        - system
+        - assistant
+        - function
+        - user
+        - elastic
+    FunctionCall:
+      type: object
+      description: Details of the function call within the message.
+      properties:
+        name:
+          type: string
+          description: The name of the function.
+        trigger:
+          type: string
+          enum:
+            - Assistant
+            - user
+            - elastic
+          description: The trigger of the function call.
+        arguments:
+          type: string
+          description: The arguments for the function call.
+      required:
+        - name
+        - trigger
+    Function:
+      type: object
+      properties:
+        name:
+          type: string
+          description: The name of the function.
+        description:
+          type: string
+          description: The description of the function.
+        parameters:
+          type: object
+          description: The parameters of the function.
+    Instruction:
+      oneOf:
+        - type: string
+          description: A simple instruction represented as a string.
+        - type: object
+          description: A detailed instruction with an ID and text.
+          properties:
+            id:
+              type: string
+              description: A unique identifier for the instruction.
+            text:
+              type: string
+              description: The text of the instruction.
+          required:
+            - id
+            - text
+    completeChatResponseExample:
+      summary: Get chat completion from the Observability AI Assistant
+      value:
+        - type: chatCompletionMessage
+          id: 'message-id'
+          message:
+            content: 'sample output'
+        - type: messageAdd
+          id: 'message-id'
+          message:
+            '@timestamp': '2025-01-17T15:45:46.255Z'
+            message:
+              content: 'sample output'
+              function_call:
+                name: 'function name'
+                arguments: 'function arguments'
+                trigger: 'message trigger'
+              role: 'message role'
+        - conversation:
+            title: 'my title'
+            id: 'message-id'
```

Should this be `conversation-id`?

---

I believe it's id? 
This is what I get when I inspect the response of the complete API in the browser:
```
{"conversation":{"title":"Checking for Alerts","id":"12c1bf9d-12b3-4e8d-b325-e2a0533e5e0f","last_updated":"2025-06-18T13:52:49.451Z"},"type":"conversationCreate"}
```

---

No, I meant the value. `message-id` doesn't seem relevant to the conversation ID

```suggestion
            id: 'conversation-id'
```

---

Ah yeah sorry! I'll change that